### DISCUSSION TITLE
Adding to list resets infinite scroll in Android app

### DISCUSSION THREAD

**Discussion by gdtyra** [community/community#102208] [Other Feature Feedback, Questions, & Ideas]
In the starred repositories list view in the Android app, if I scroll beyond the first page and add a repository to a list via the 3-dot menu then my infinite scroll resets to the initial page of repositories and I have to scroll while waiting for page loads again.

(wanted to label this as a bug but maybe I can't label from the Android app?)

**Comment by Al3676** [comment]


**Reply by Charlottedubois19772** [reply]
Is there codes to get my email that I have been locked out of can someone help me get my into my forgot password to my Gmail email I forgot the password in November.  It's a 8 digital code.

**Reply by Al3676** [reply]
Seated flawer more

**Comment by Kingcb1** [comment]
> Thankyou

**Comment by Charlottedubois19772** [comment]
What the answer???

**Comment by Charlottedubois19772** [comment]
Great! Thank you for you stupid reply

**Comment by YifanLiang-hust** [comment]
=\
=\

**Comment by nguyenvansi998** [comment]
tiêng việt nam
